config/eslint.json

```diff
@@ -13,7 +13,8 @@
     "OSM": "writable",
     "Matomo": "readonly",
     "Qs": "readonly",
-    "updateLinks": "readonly"
+    "updateLinks": "readonly",
+    "Turbo": "writable"
```

Comment:
"readonly" works fine here, you're not going to rewrite `Turbo` anywhere in js

Reply:
He is writing to `Turbo.session.drive` to make turbo opt-in rather than opt-out.

Reply:
He's not rewriting `window.Turbo`. That's what eslint checks.

Reply:
> That's what eslint checks.

I'll look at this and will adjust the config if `readonly` suffices 👍🏽

Reply:
I think it does. Locally `eslint` did not complain with `readonly`.